src/TTextEdit.cpp

```diff
@@ -60,6 +60,7 @@ TTextEdit::TTextEdit(TConsole* pC, QWidget* pW, TBuffer* pB, Host* pH, bool isLo
 , mIsLowerPane(isLowerPane)
 , mLastRenderBottom(0)
 , mMouseTracking(false)
```

Comment:
🤔 Do we still need `mMouseTracking` now - or is it replaceable throughout this method with `mMouseTrackLevel` - with the latter being non-zero (logically `true` as far as C/C++ goes) being the same as the former being `true`?

Reply:
We do, yeah.  The issue is: what happens after we release the mouse button the first time?  We don't want to track at that point (since the mouse is released, and we may never press again).  But we need to record somewhere that we've started up the trackLevel, so that if we press the mouse again it counts as a double/triple click.

So `mMouseTracking` is unset on `mouseReleaseEvent`, but `mMouseTrackLevel` is not.